api-gateway/src/main/kotlin/com/saveourtool/save/gateway/service/BackendService.kt

```diff
@@ -64,9 +63,6 @@ class BackendService(
     @Suppress("UnusedParameter")
     fun findByPrincipal(principal: Principal, session: WebSession): Mono<SaveUserDetails> = when (principal) {
         is OAuth2AuthenticationToken -> findByOriginalLogin(principal.authorizedClientRegistrationId, principal.name)
-            .switchIfEmptyToResponseException(HttpStatus.INTERNAL_SERVER_ERROR) {
```

Comment:
?

Reply:
it's leftover after hotfix. `findByOriginalLogin` returns error when response is empty from backend. `INTERNAL_SERVER_ERROR` was for internal error when attribute is missed in session